mmdet/models/dense_heads/rpn_test_mixin.py

```diff
@@ -1,10 +1,7 @@
-import logging
```

Comment:
This file can be moved to `models/dense_heads/`.

Reply:
File is already moved to ``models/dense_heads/``